src/rebar3_eqc.erl

```diff
@@ -311,7 +311,7 @@ compile_tests(State, TestApps, Suites, RawOpts) ->
                 TopAppsPaths = app_paths(NewState),
                 rebar_utils:update_code(rebar_state:code_paths(NewState, all_deps)
                                         -- TopAppsPaths, [soft_purge]),
-                code:add_patha(TopAppsPaths),
+                code:add_pathsa(TopAppsPaths),
```

Comment:
I see this is merged in master as https://github.com/kellymclaughlin/rebar3-eqc-plugin/commit/00658c7